easybuild/easyconfigs/g/GCC/GCC-4.9.3-binutils-2.25.eb

```diff
@@ -25,7 +26,12 @@ sources = [
     'mpc-1.0.2.tar.gz',
 ]
 
-builddependencies = [('binutils', binutilsver)]
+dependencies = [
+    # GCC built on top of (dummy-built) binutils
+    ('GCCcore', gccver, '-binutils-%s' % binutilsver),
```

Comment:
Yeah, I can clean this out. This is a hangover from the toolchain-based naming scheme that I use. With the latest release I can improve the intelligence of the naming scheme so this isn't necessary.